jobs/dynatrace-oneagent/templates/start-oneagent.sh.erb

```diff
@@ -39,7 +39,13 @@ setWatchdogPid() {
 checkForInstaller
 
 if ! runServiceCommand start; then
-    exit 1
+    echo "Couldn't start OneAgent service, trying to repair environment by re-running installation procedure"
```

Comment:
Any restart semantics for this command? i.e., if something is broken, does it get called in a loop? Is that problematic?

Reply:
I intentionally kept this simple since it's just a shell script :-)
Even if it loops for some weird reason, bosh will kill it once the whole process at some point when it's timing out.
https://bosh.io/docs/job-lifecycle/#start